android/app/src/androidTest/java/com/microsoft/codepush/common/managers/SettingManagerAndroidTests.java

```diff
@@ -115,9 +115,9 @@ public void pendingUpdateIsNull() throws Exception {
     @Test
     public void failedUpdateCompatibilityTest() throws Exception {
         mSettingsManager.removeFailedUpdates();
-        CodePushLocalPackage codePushLocalPackage = createLocalPackage(PACKAGE_HASH);
+        CodePushPackage codePushLocalPackage = createPackage(PACKAGE_HASH);
```

Comment:
Why we have changed `CodePushLocalPackage` to `CodePushPackage`?

Reply:
@sergey-akhalkov because that was the logical mistake I previously fixed in the main code, but didn't fix in tests.